pkg/action/scan.go

```diff
@@ -324,7 +324,7 @@ func recursiveScan(ctx context.Context, c malcontent.Config) (*malcontent.Report
 
 		// Add the sorted paths and file reports to the parent report and render the results
 		for _, k := range pathKeys {
-			finding, ok := scanPathFindings.Load(k)
+			finding, ok := scanPathFindings.LoadAndDelete(k)
```

Comment:
It took a while to track this down but luckily this is all that was needed to avoid needing to loop through each scan path one at a time in the base command code in `malcontent.go`.